From 9e11f34368a8fa8fa4e25f56ac7737f2e9f47d65 Mon Sep 17 00:00:00 2001
From: Rob <support@risingedgeindustries.com>
Date: Sat, 9 Jul 2022 14:26:30 -0700
Subject: [PATCH] added new line to cmd/bootmenu.c

---
 cmd/bootmenu.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cmd/bootmenu.c b/cmd/bootmenu.c
index 7d8b78c1b1..3b7c31d2ab 100644
--- a/cmd/bootmenu.c
+++ b/cmd/bootmenu.c
@@ -88,8 +88,8 @@ static void bootmenu_autoboot_loop(struct bootmenu_data *menu,
 
 	if (menu->delay > 0) {
 		printf(ANSI_CURSOR_POSITION, menu->count + 5, 1);
-		printf("  Hit any key to stop autoboot: %2d ", menu->delay);
-		printf("--> Rob test message!");
+		printf("  Hit any key to stop autoboot: %2d \n", menu->delay);
+		printf("--> Rob test message!\n");
 	}
 
 	while (menu->delay > 0) {
